src/scss/components/_media-element.scss

```diff
@@ -144,11 +144,11 @@ $mediaElement-spacing           : spacing(x2);
 
     .c-mediaElement-text {
         margin-top: 0;
-        @include font-size(small, false);
+        @include font-size(caption, false);
     }
 
     .c-mediaElement-text--large {
-        @include font-size(large, false);
+        @include font-size(heading-m, false);
```

Comment:
Semantically, large text isn't necessarily a heading - maybe `body-xl` would be a good addition - I know there might be some duplication, but might be a bit clearer.

Reply:
I agree it might be clearer to devs when styling components which font size to use if we add the extra sizes.

Reply:
Agree – atm, this is just a lift and shift to replace the old variables with equivalent new ones from the global design tokens. As design go through each component when working on PIE, we should be able to replace these with alias variable tokens specific to each component rather than having these point at the defined heading tokens (just as that's all we have atm, they haven't looked at specific components like the `mediaElement` component yet).

I'll add a TODO comment to any of these that seem to be un-semantic so that we can make sure they get looked at (these will get ported out of fozzie as they move into the `fozzie-components` repo).

Reply:
Sounds good to me 👍